x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/connector_types/bedrock.ts

```diff
@@ -407,6 +409,47 @@ export default function bedrockTest({ getService }: FtrProviderContext) {
               data: { message: bedrockSuccessResponse.completion },
             });
           });
+
+          it.only('should invoke stream with assistant AI body argument formatted to bedrock expectations', async () => {
+            await new Promise<void>((resolve, reject) => {
+              const receivedChunks: any[] = [];
+
+              const passThrough = new PassThrough();
+
+              supertest
+                .post(`/api/actions/connector/${bedrockActionId}/_execute`)
+                .set('kbn-xsrf', 'foo')
+                //   .set('Accept', '*/*')
+                //   .set('Accept-Encoding', 'chunked')
+                //   .set('Content-Type', 'application/json;charset=UTF-8')
+                .on('error', reject)
+                .send({
+                  params: {
+                    subAction: 'invokeStream',
+                    subActionParams: {
+                      messages: [
+                        {
+                          role: 'user',
+                          content: 'Hello world',
+                        },
+                      ],
+                    },
+                  },
+                })
+                .pipe(passThrough);
+
+              passThrough.on('data', (chunk) => {
+                // logs the state of the stream, but not the stream itself
+                console.log('passThrough.data???', chunk.toString());
```

Comment:
Only returns the state of the stream, not the stream itself

```
passThrough.data??? {
   "status":"ok",
   "data":{
      "_readableState":{
         "objectMode":false,
         "highWaterMark":16384,
         "buffer":{
            "head":null,
            "tail":null,
            "length":0
         },
         "length":0,
         "pipes":[
            {
               "_readableState":{
                  "objectMode":false,
                  "highWaterMark":16384,
                  "buffer":{
                     "head":null,
                     "tail":null,
                     "length":0
                  },
                  "length":0,
                  "pipes":[
                     
                  ],
                  "flowing":true,
                  "ended":false,
                  "endEmitted":false,
                  "reading":false,
                  "constructed":true,
                  "sync":false,
                  "needReadable":false,
                  "emittedReadable":false,
                  "readableListening":false,
                  "resumeScheduled":true,
                  "errorEmitted":false,
                  "emitClose":true,
                  "autoDestroy":true,
                  "destroyed":false,
                  "errored":null,
                  "closed":false,
                  "closeEmitted":false,
                  "defaultEncoding":"utf8",
                  "awaitDrainWriters":null,
                  "multiAwaitDrain":false,
                  "readingMore":false,
                  "dataEmitted":false,
                  "decoder":null,
                  "encoding":null
               },
               "_events":{
                  "error":[
                     null,
                     null
                  ],
                  "close":[
                     null,
                     null
                  ],
                  "finish":[
                     null,
                     null
                  ]
               },
               "_eventsCount":7,
               "_writableState":{
                  "objectMode":false,
                  "highWaterMark":16384,
                  "finalCalled":false,
                  "needDrain":false,
                  "ending":false,
                  "ended":false,
                  "finished":false,
                  "destroyed":false,
                  "decodeStrings":true,
                  "defaultEncoding":"utf8",
                  "length":0,
                  "writing":false,
                  "corked":0,
                  "sync":true,
                  "bufferProcessing":false,
                  "writecb":null,
                  "writelen":0,
                  "afterWriteTickInfo":null,
                  "buffered":[
                     
                  ],
                  "bufferedIndex":0,
                  "allBuffers":true,
                  "allNoop":true,
                  "pendingcb":0,
                  "constructed":true,
                  "prefinished":false,
                  "errorEmitted":false,
                  "emitClose":true,
                  "autoDestroy":true,
                  "errored":null,
                  "closed":false,
                  "closeEmitted":false
               },
               "allowHalfOpen":true
            }
         ],
         "flowing":true,
         "ended":false,
         "endEmitted":false,
         "reading":false,
         "constructed":true,
         "sync":false,
         "needReadable":false,
         "emittedReadable":false,
         "readableListening":false,
         "resumeScheduled":true,
         "errorEmitted":false,
         "emitClose":true,
         "autoDestroy":true,
         "destroyed":false,
         "errored":null,
         "closed":false,
         "closeEmitted":false,
         "defaultEncoding":"utf8",
         "awaitDrainWriters":null,
         "multiAwaitDrain":false,
         "readingMore":false,
         "dataEmitted":false,
         "decoder":null,
         "encoding":null
      },
      "_events":{
         
      },
      "_eventsCount":7,
      "_writableState":{
         "objectMode":false,
         "highWaterMark":16384,
         "finalCalled":false,
         "needDrain":false,
         "ending":false,
         "ended":false,
         "finished":false,
         "destroyed":false,
         "decodeStrings":true,
         "defaultEncoding":"utf8",
         "length":0,
         "writing":false,
         "corked":0,
         "sync":true,
         "bufferProcessing":false,
         "writecb":null,
         "writelen":0,
         "afterWriteTickInfo":null,
         "buffered":[
            
         ],
         "bufferedIndex":0,
         "allBuffers":true,
         "allNoop":true,
         "pendingcb":0,
         "constructed":true,
         "prefinished":false,
         "errorEmitted":false,
         "emitClose":true,
         "autoDestroy":true,
         "errored":null,
         "closed":false,
         "closeEmitted":false
      },
      "allowHalfOpen":true
   },
   "connector_id":"c76a64e3-7382-4f83-a525-703f3f0ad6d2"
}
```

This is what the response looks like logged in real life:
```
{
  status: 'ok',
  data: Transform {
    _readableState: ReadableState {
      objectMode: false,
      highWaterMark: 16384,
      buffer: BufferList { head: null, tail: null, length: 0 },
      length: 0,
      pipes: [Array],
      flowing: true,
      ended: false,
      endEmitted: false,
      reading: false,
      constructed: true,
      sync: false,
      needReadable: false,
      emittedReadable: false,
      readableListening: false,
      resumeScheduled: true,
      errorEmitted: false,
      emitClose: true,
      autoDestroy: true,
      destroyed: false,
      errored: null,
      closed: false,
      closeEmitted: false,
      defaultEncoding: 'utf8',
      awaitDrainWriters: null,
      multiAwaitDrain: false,
      readingMore: false,
      dataEmitted: false,
      decoder: null,
      encoding: null,
      [Symbol(kPaused)]: false
    },
    _events: [Object: null prototype] {
      prefinish: [Function: prefinish],
      unpipe: [Function: onunpipe],
      error: [Function: onerror],
      close: [Function],
      finish: [Function],
      end: [Function],
      data: [Function: ondata]
    },
    _eventsCount: 7,
    _maxListeners: undefined,
    _writableState: WritableState {
      objectMode: false,
      highWaterMark: 16384,
      finalCalled: false,
      needDrain: false,
      ending: false,
      ended: false,
      finished: false,
      destroyed: false,
      decodeStrings: true,
      defaultEncoding: 'utf8',
      length: 0,
      writing: false,
      corked: 0,
      sync: true,
      bufferProcessing: false,
      onwrite: [Function: bound onwrite],
      writecb: null,
      writelen: 0,
      afterWriteTickInfo: null,
      buffered: [],
      bufferedIndex: 0,
      allBuffers: true,
      allNoop: true,
      pendingcb: 0,
      constructed: true,
      prefinished: false,
      errorEmitted: false,
      emitClose: true,
      autoDestroy: true,
      errored: null,
      closed: false,
      closeEmitted: false,
      [Symbol(kOnFinished)]: []
    },
    allowHalfOpen: true,
    _transform: [Function: transform],
    [Symbol(kCapture)]: false,
    [Symbol(kCallback)]: null
  },
  actionId: '337fc345-352e-459f-8c2c-817604ad14bd'
}
```

As you can see, on my real server, I get a Transform. On `supertest`, i get only the state of the Tranform